app/src/main/java/de/vectordata/skynet/ui/base/SkynetActivity.java

```diff
@@ -46,6 +53,19 @@ public void onAuthenticationFailed(AuthenticationFailedEvent event) {
             Dialogs.showMessageBox(this, R.string.error_header_connect, R.string.error_invalid_session);
     }
 
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onSyncFinished(SyncFinishedEvent event) {
+        int corruptedMessages = SkynetContext.getCurrent().getNetworkManager().getLastSyncCorruptedMessages();
+        if (corruptedMessages > 0) {
+            new AlertDialog.Builder(this)
+                    .setTitle(R.string.error_header_sync)
+                    .setMessage(getString(R.string.error_corrupted_messages, corruptedMessages))
+                    .setPositiveButton(R.string.ok, null)
+                    .setNeutralButton(R.string.action_report_issue, (dialog, which) -> startBrowser("https://www.skynet.app/bug"))
```

Comment:
Good idea to make reporting as easy as possible but without any further information these reports will be useless for us. Maybe you can open a url like `https://www.skynet.app/bug?title=…&description=…` and mention the affected message IDs. We could look them up in the server database and might be able to find out which packets are often affected.

Reply:
That would also be an extension to the web app...

Reply:
You can also show the message ID in the dialog.

For the extension I can open a new feature at [skynet-im/skynet-website](https://github.com/skynet-im/skynet-website). Have you uploaded the source code of this web app?

Reply:
No, it's not uploaded because the webapp is only a temporary app that uses a GitHub API key to make Issues so that betatesters don't have to have a GitHub account

Reply:
This problem is now tracked by #32.